### DISCUSSION TITLE
Change to "Virtual devices folder" in preferences is ignored

### DISCUSSION THREAD

**Discussion by bumper314** [Genymobile/feedback#48] [Genymotion Desktop] [answered]
I just did a fresh install of Genymotion 3.3.2 on macOS 10.14.  The first thing I did was go into the Preferences and change the "Virtual devices folder" path to another disk with more spaces.  I then added a new virtual device, and it was promptly installed to the original path (~/.Genymobile/Genymotion/deployed), which is precisely what I did not want.

![Screen Shot 2023-02-15 at 16 32 11](https://user-images.githubusercontent.com/43441/219216641-413744de-6e42-41b4-a483-6ddbeb2d9a48.png)

**Comment by jmaneyrol69** [comment, marked as answer]
@bumper314 We have reproduced the problem, but you can workaround it by following these steps:
1. Change the Virtual devices folder
2. Close Genymotion and re-start it

Then it should work as expected.
